File: analyst/docs/competitors/pascal/CircleSquare/CircleSquare.pas
program CircleSquare;
const Pi = 3.14159;
var 
  radius, area: real;
begin
  writeln('Enter the radius of the circle:');
  read(radius);
  if radius < 0 
  then
    begin
      writeln('Error: Radius cannot be negative.');
    end
  else
    begin
      area := Pi * radius * radius;
      writeln('Area of the circle: ', area:0:2);
    end;
  readln;
end.

File: docs/competitors/c++/CircleSquare.cpp
#include <iostream> 
#include <iomanip> 

int main() { 
	const double PI = 3.14159;
	double radius; 
	std::cout << "Enter the radius of the circle: "; 
	std::cin >> radius; 
	if (radius < 0) { 
		std::cout << "Error: Radius cannot be negative." << std::endl; 
	} 
	else { 
		double area = PI * radius * radius;
		std::cout << "Area of the circle: " << std::fixed << std::setprecision(2) << area << std::endl; 
	} 
return 0; 
}